src/pal/prebuilt/idl/clrdata_i.cpp

```diff
@@ -87,5 +87,13 @@ MIDL_DEFINE_GUID(IID, IID_ICLRDataEnumMemoryRegions,0x471c35b4,0x7c2f,0x4ef0,0xa
 }
 #endif
 
-
+#if !(defined(_MSC_VER) || defined(__llvm__))
+BIND_UUID_OF(ICLRDataTarget)
```

Comment:
Maybe `MIDL_DEFINE_GUID(IID, IID_XXX, ...`, can become `MIDL_DEFINE_GUID(IID, IID, XXX, ..` and this `MIDL_DEFINE_GUID` is defined as:

```c
#define MIDL_DEFINE_GUID(type,typePrefix,name,l,w1,w2,b1,b2,b3,b4,b5,b6,b7,b8)         \
        const type name = {type,typePrefix_##w1,w2,{b1,b2,b3,b4,b5,b6,b7,b8}};      \
        BIND_UUID_OF(w1, typePrefix)
```

For non-GNUC path, `BIND_UUID_OF` could be defined blank, to avoid ifdef in many places.